tools/codegen/api/autograd.py

```diff
@@ -71,6 +97,9 @@ class DifferentiabilityInfo:
     # The derivatives formulae for this function.
     derivatives: Sequence[Derivative]
 
+    # The forward derivatives formulae for this function.
+    forward_derivatives: Sequence[ForwardDerivative]
```

Comment:
It might be useful to point out that `derivatives` length is the number of input, but `forward_derivatives` length is the number of output